doozer/doozerlib/backend/konflux_client.py

```diff
@@ -432,6 +447,12 @@ def _modify_param(params: List, name: str, value: Union[str, bool, list[str]]):
                 }]
             }]
 
+        if not sast:
+            params = self._delete_param(params, "sast-unicode-check")
+            params = self._delete_param(params, "sast-shell-check")
```

Comment:
if we want to filter out these params, would recommend something like this:
```suggestion
        if not sast:
            sast_params = ("sast-unicode-check", "sast-shell-check")
            params = [p for p in params if p.get("name") not in sast_params]
```